templates/default/systemd/sensu-api.service.erb

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=sensu api
+
+[Service]
+User=sensu
+Group=sensu
+EnvironmentFile=/etc/default/sensu
+ExecStart=/opt/sensu/bin/sensu-api -c /etc/sensu/config.json -d /etc/sensu/conf.d -e /etc/sensu/extensions -l /var/log/sensu/sensu-api.log
```

Comment:
@cwjohnston should i now use the sensu-service binstub? Something like `ExecStart=/opt/sensu/bin/sensu-service api` then strip the parameters and set them in the environment file instead as env vars?